recipe-box-backend/src/main/java/com/endava/recipebox/service/impl/RecipeServiceImpl.java

```diff
@@ -98,4 +99,36 @@ public String createRecipe(RecipeAddRequestDTO recipeAddRequestDTO, Long userId)
 
         return "Recipe added successfully";
     }
+
+    @Override
+    @Transactional
+    public String editRecipe(RecipeEditRequestDTO recipeAddRequestDTO, Long userId){
```

Comment:
rename to update recipe. The edit operation is taking place in frontend side, in backend we just update values